packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -892,11 +925,11 @@ const service = new ecs.Ec2Service(stack, 'Service', {
 You can enable sending logs of your execute session commands to a CloudWatch log group or S3 bucket by configuring
 the `executeCommandConfiguration` property for your cluster. The default configuration will send the
 logs to the CloudWatch Logs using the `awslogs` log driver that is configured in your task definition. Please note,
-when using your own `logConfiguration` the log group or S3 Bucket specified must already be created. 
```

Comment:
Please revert formatting changes. We generally discourage mixing these in the same PR as it makes it hard to discern what actually changed.

Feel free to submit another pull request with formatting fixes.